src/v2/ion/ViewClassNamesFactory.js

```diff
@@ -6,6 +6,9 @@ const FORMNAME_CLASSNAME_MAPPINGS = {
     [FORMS.IDENTIFY]: 'primary-auth',
     'password': 'primary-auth'
   },
+  [FORMS.IDENTIFY_RECOVERY]: {
+    [FORMS.IDENTIFY_RECOVERY]: 'forgot-password',
```

Comment:
unsure of this class. This is not really the forgot password screen, but the identifier view in the forgot password flow

Reply:
we are using forgot-password in v1 too. lol

![Screen Shot 2021-02-02 at 3 01 16 PM](https://user-images.githubusercontent.com/17267130/106674093-8443ba00-6567-11eb-9ed8-f919e669ce88.png)

Reply:
Yea, this looks right then.. my understanding of ViewClassNamesFactory.js is that we should add the same classes to the oie view as the corresponding view in v1. I wasn't sure which v1 view mapped to the IDX view, but this makes sense